controllers/comment.controllers.js

```diff
@@ -36,4 +39,21 @@ exports.postComment = (req, res, next)=>{
     .catch((err)=>{
         next(err)
     })
+}
+
+exports.deleteCommentById = (req, res, next)=>{
+    const {comment_id} = req.params;
+
+    const commentExistence = checkCommentExist(comment_id);
+    const deleteCommentQuery = removeCommentById(comment_id);
+    const queries = [deleteCommentQuery, commentExistence];
+
+    Promise.all(queries)
+    .then(()=>{
+        res.status(204).send()
+    })
+    .catch((err)=>{
+        console.log(err)
```

Comment:
Don't forget to remove console logs once you've finished developing on an endpoint 🧹